app/post_upgrade.go

```diff
@@ -0,0 +1,184 @@
+package app
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/blang/semver"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/rancher/longhorn-manager/datastore"
+	longhorn "github.com/rancher/longhorn-manager/k8s/pkg/apis/longhorn/v1alpha1"
+	"github.com/rancher/longhorn-manager/types"
+)
+
+const (
+	FlagFromVersion = "from-version"
+	FlagToVersion   = "to-version"
+)
+
+var ownerKindReplica = longhorn.SchemeGroupVersion.WithKind("Replica").String()
+
+func PostUpgradeCmd() cli.Command {
+	return cli.Command{
+		Name: "post-upgrade",
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  FlagFromVersion,
+				Usage: "Specify version we upgraded from",
+			},
+			cli.StringFlag{
+				Name:   FlagToVersion,
+				Value:  VERSION,
+				Usage:  "Specify version we upgraded to",
+				Hidden: true,
+			},
+			cli.StringFlag{
+				Name:  FlagKubeConfig,
+				Usage: "Specify path to kube config (optional)",
+			},
+			cli.StringFlag{
+				Name:   FlagNamespace,
+				EnvVar: types.EnvPodNamespace,
+			},
+		},
+		Action: func(c *cli.Context) {
+			logrus.Infof("Running post-upgrade...")
+			defer logrus.Infof("Completed post-upgrade.")
+
+			if err := postUpgrade(c); err != nil {
+				logrus.Fatalf("Error during post-upgrade: %v", err)
+			}
+		},
+	}
+}
+
+func parseVersion(version string) (semver.Version, error) {
+	version = strings.TrimPrefix(version, "v")
+	return semver.Make(version)
```

Comment:
We're already using "k8s.io/kubernetes/pkg/util/version" in https://github.com/rancher/longhorn-manager/blob/master/app/driver.go . Can we reuse it without introducing another dependency?